dat/ships/pirate/dealsealer.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<ship name="Dealsealer">
```

Comment:
I feel like this variant probably needs a bit more of an interesting twist. It feels a bit bland at the moment.

Reply:
Could just not include it. I agree it's boring